tools/cbmc/proofs/DNS/DNSgetHostByName/Makefile.json

```diff
@@ -0,0 +1,23 @@
+{
+  "ENTRY": "DNSgetHostByName",
+  ################################################################
+	# This configuration flag sets callback to 0
+  "callback": 0,
+  "CBMCFLAGS": 
+  [
+  		"--unwind 1",
+  		"--unwindset prvProcessDNSCache.0:5,prvGetHostByName.0:11,prvCreateDNSMessage.0:11,prvCreateDNSMessage.1:11,strlen.0:11,__builtin___strcpy_chk.0:11,strcmp.0:11",
```

Comment:
Same comment as [this](https://github.com/aws/amazon-freertos/pull/663#discussion_r288143110)